### PR TITLE
Issue #321- Paired mr us brain demo

### PR DESCRIPTION
**Pull request by RemiDelaunay**
# Description

Review of the paired_mrus_brain demo.

# Changes

I corrected typos and modified description of the demo. I did not have any problem running the demo.

New edits:
Both suggestions (see below) incorporated:
- typo in dataset corrected
- message displayed at the end of predict to point the user to the folder containing the generated and original images

# Suggestion

- I don't know if it worth mentioning this but there is a typo for the dataset name. All the folder created refers to "respect-dataset" instead of "resect-dataset". I didn't correct the typo to avoid any confusion because the directory from which the data is downloaded (https://github.com/ucl-candi/dataset_respect/archive/master.zip) have the same typo.

- The demo only outputs a plot is in 2D and it is difficult to visualize/assess the registration process. The visualization of the data could be improved by exporting the warped images/DDF in a dedicated folder.

### CODE REVIEW

**Comment by s-sd** [comment]
@RemiDelaunay, thanks for reviewing the demo. Is it okay if I use this branch to correct the "dataset_respect" typo?

**Comment by RemiDelaunay** [comment]
@s-sd Yes it is ok :)

**Comment by RemiDelaunay** [comment]
@s-sd Thank you for your help. Can I assign the PR for review ?

**Comment by s-sd** [comment]
@RemiDelaunay, sorry I changed this back to WIP. I'm just re-running both the full version and reduced dataset version of the demo to see that it runs fine. I'll let you know once I've updated everything :)

**Comment by RemiDelaunay** [comment]
@s-sd sorry for rushing ;)

**Comment by s-sd** [comment]
@RemiDelaunay, I've checked and they run fine with the name change. I've tried to address both your suggestions. So the two additional things I've done are:
- correct typo in dataset directory name
- add a message that points to all the images generated by predict (these include the warped and original images)

I'll add these two changes to the description of this PR as well

Please feel free to remove 'WIP' if there are no further changes

File: demos/paired_mrus_brain/README.md [hunk 1]
```diff
@@ -5,23 +5,22 @@
 **NOTE**
 
 Please read the
-[DeepReg Demo Disclaimer](https://github.com/DeepRegNet/DeepReg/blob/master/demos/README.md).
+[DeepReg Demo Disclaimer](https://deepreg.readthedocs.io/en/325-improve-contributing-pages/demo/introduction.html#demo-disclaimer).
```
Comment (mathpluscode) [inline demos/paired_mrus_brain/README.md:8]:
```
[DeepReg Demo Disclaimer](introduction.html#demo-disclaimer).
```

Sorry we might have some error earlier the disclaimer link should be like above

**Review by mathpluscode — CHANGES_REQUESTED** [review]
Please
- Fix the disclaimer link
- merge master to your branch and solve conflicts

File: demos/paired_mrus_brain/README.md (hunk 1 above)
Comment (RemiDelaunay) [reply demos/paired_mrus_brain/README.md]: @mathpluscode I just fixed the link and the branch is up-to-date with the master branch.

**Review by mathpluscode — APPROVED** [review]

**Pull request merged**